src/debugger/core.clj

```diff
@@ -22,6 +22,37 @@
        (println '~x "->" x#)
        x#)))
 
+
+(def ^{:private true} registered-breakpoints (atom #{}))
```

Comment:
afair `#{}` doesn't guarantee order. 
besides, while connect you always using `(vec @registered-breakpoints)`
and functions will be different objects (all unique).
so, just use `[]`